bundles/pixi.js-webworker/src/adapter.ts

```diff
@@ -0,0 +1,23 @@
+import { settings } from '@pixi/settings';
+
+import type { IAdapter } from '@pixi/settings';
```

Comment:
We should import from core instead. settings/constants/ticker/utils/runner are all internal packages only to core.

```suggestion
import { settings } from '@pixi/core';

import type { IAdapter } from '@pixi/core';
```